### DISCUSSION TITLE
next release

### DISCUSSION THREAD

**Discussion by dbluelle** [lightspark/lightspark#638] [General]
Hi everybody,

I'm planning on releasing a new version of lightspark next weekend, if nobody has any objections.

**Comment by huwdp** [comment]
My PR is not ready so it'll have to come after the release. No objections here

**Comment by dbluelle** [comment]
OK, new release (0.8.4) is online.

**Comment by dbluelle** [comment]
I've made a new release (0.8.4.1) which includes just a hotfix for Windows, as 0.8.4 crashes on Windows for a lot of swf files

**Reply by Pancake689** [reply]
Mine still crashes, though I'm not sure I use it correctly. Version 8.3 worked, but poorly. 8.4 and 8.4.1 both crash upon launching or loading the file.

**Reply by huwdp** [reply]
Hi there, if you have any bad then please send us an issue via github

**Comment by dbluelle** [comment]
OK, I think it's time for a new release.
So if nobody objects and no big new issues arise, I'm planning to release version 0.8.5 in two weeks.

**Reply by huwdp** [reply]
I think the new release will be impressive! Looks good! :D

**Reply by eiglow** [reply]
Would it be a good idea to have something like a release candidate branch?

**Reply by dbluelle** [reply]
I don't think that is neccessary as I'm not planning on committing any big changes until the release, so it's just bug fixes until then.

**Comment by dbluelle** [comment]
Release 0.8.5 is online.

**Comment by dbluelle** [comment]
If no big new issues arise I will do a new release in two weeks.

**Reply by dbluelle** [reply]
Release 0.8.6 is online.

**Comment by dbluelle** [comment]
I forgot to mention it here, hotfix release 0.8.6.1 is available.
This release has fixes for Windows, no relevant changes for Linux

**Comment by dbluelle** [comment]
I think it's time for a new release, so I'm planning to do that next weekend.
Oh, and in case you are on Windows and haven't seen it, we now have development releases based on the latest commit, see https://github.com/lightspark/lightspark/releases/tag/latest

**Reply by dbluelle** [reply]
new release 0.8.7 is online

**Comment by dbluelle** [comment]
release 0.9 is planned for next weekend

**Reply by dbluelle** [reply]
release 0.9.0 is online

**Comment by dbluelle** [comment]
release 0.10 is planned for next weekend.
